tests/ssg_test_suite/common.py

```diff
@@ -233,29 +248,155 @@ def _make_file_root_owned(tarinfo):
     return tarinfo
 
 
-def create_tarball():
+def _rel_abs_path(current_path, base_path):
```

Comment:
What is the benefit of resolving the examined path to the absolute path before the comparison?

Reply:
Sorry, could you expand on this? Are you asking why call `abspath` prior to `relpath`? My understanding was the supplied path must have a path with prefix `base_path`, so the `abspath` ensures that we have a correctly-rooted path structure.

The intent of this is to make copying into the tarball correct, with subdirectories (like `tests/ocp4/...`).

Reply:
Ah I see now. This will be addressed in #7211.